src/components/header/Header.jsx

```diff
@@ -0,0 +1,50 @@
+import { useLocation } from "react-router-dom";
+import { useState } from "react";
+import {
+  AuthIcon,
+  BarsIcon,
+  CartIcon,
+  FilterIcon,
+  LightDarkIcon,
+  WishListIcon,
+} from "./icons";
+import { Logo } from "./logo/Logo";
+import { Search } from "./search/Search";
+
+export function Header() {
+  const currentPath = useLocation();
+
+  const [showMobileIcons, setShowMobileIcons] = useState(false);
+  return (
+    <header className="sm-header">
+      <nav className="sm-nav-new">
+        <Logo />
+
+        {currentPath.pathname === currentPath.pathname ? (
```

Comment:
here you are equating the same thing, check it, you might be trying to equate current pathname with some other pathname

Reply:
I got confused aswell.

Reply:
this is for the test , i will be updating it :)